drv/riscv-plic-server/build.rs

```diff
@@ -93,13 +93,13 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
         let int_num: InterruptNum = InterruptNum(*irq as u32);
 
-        let notification: u32 = TASK_CONFIG.notification[i];
+        let notif: u32 = TASK_CONFIG.notification[i];
 
-        irq_task_map.push((int_num, (task_id, notification)));
+        irq_task_map.push((int_num, (task_id, notif)));
 
         let owner: InterruptOwner = InterruptOwner {
             task: task_id.index() as u32,
-            notification: notification,
+            notification: notif,
```

Comment:
nit: Rather than changing the other places to use `notif`, it would have been simpler to just change this line to drop the `: notification` part, right?

Reply:
That's what Clippy suggested too, but I think that from a style perspective it makes more sense for the struct members to be assigned in a uniform manner, either all implicitly or explicitly. And in this case it was simplest to make it all work explicitly. This is purely aesthetic, however, so I'm fine with whatever.

Reply:
I think it's fine. Just wanted to understand the thinking behind going this way :smile: or if there was some limitation because of which it couldn't be done.